### PR TITLE
update insert function of database

### PR DESCRIPTION
**Pull request by SoniaChoo**
update insert function of the database
in order to insert the corresponding info into the database

### CODE REVIEW

File: db/dish.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// InsertDish is to insert a dish info into database
+func InsertDish(d model.Dish) error {
+	db, err := DBFactory()
+	if err != nil {
+		log.Printf("error connect database, %v\n", err)
+	}
+	// start to execute SQL query
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+	defer cancel()
+	_, err = db.QueryContext(ctx, "insert into user(dish_id, rest_id, price, dish_name, stock, slaes, creat_time) values(?,?,?,?,?,?,?)",
```
Comment (ziyitony) [inline db/dish.go]: please check the SQL statement in the database first.
**sales** is the wrong spell, **dish_id** is not needed cuz it's PK(primary key)

File: db/dish.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+		d.DishId, d.RestId, d.Price, d.DishName,d.Stock,d.Sales,d.CreatTime)
+	if err != nil {
+		log.Printf("record inserting with error %s\n", err.Error())
+		return err
+	}
+
+	return nil
+}
```
Comment (ziyitony) [inline db/dish.go]: DO **NOT** forget the new line at EOF

File: db/order_detail.go [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+// InsertOrderDetail is to insert a order detail info into database
+func InsertOrderDetail(od model.Order_detail) error {
+	db, err := DBFactory()
+	if err != nil {
+		log.Printf("error connect database, %v\n", err)
+	}
+	// start to execute SQL query
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+	defer cancel()
+	_, err = db.QueryContext(ctx, "insert into user(detail_id, dish_id, rest_id, order_id, price, number) values(?,?,?,?,?,?)",
```
Comment (ziyitony) [inline db/order_detail.go]: insert into **user**?

**Review by ziyitony — CHANGES_REQUESTED** [review]
too many updates needed due to same problems

File: db/dish.go (hunk 1 above)
Comment (SoniaChoo) [reply db/dish.go]: thanks, have checked and updated

File: db/order_detail.go (hunk 3 above)
Comment (SoniaChoo) [reply db/order_detail.go]: thanks, updated

File: db/dish.go (hunk 2 above)
Comment (SoniaChoo) [reply db/dish.go]: thanks, updated

**Review by ziyitony — APPROVED** [review]
what a good job!

**Pull request merged**